diff -Naur nvidia-modprobe-545.52.old/Makefile nvidia-modprobe-545.52/Makefile
--- nvidia-modprobe-545.52.old/Makefile	2024-04-02 12:47:44.381521131 +0200
+++ nvidia-modprobe-545.52/Makefile	2024-04-02 12:47:54.277591761 +0200
@@ -110,7 +110,7 @@
 .PHONY: MANPAGE_install
 MANPAGE_install: $(MANPAGE)
 	$(MKDIR) $(MANDIR)
-	$(INSTALL) $(INSTALL_BIN_ARGS) $< $(MANDIR)/$(notdir $<)
+	$(INSTALL) $(INSTALL_DOC_ARGS) $< $(MANDIR)/$(notdir $<)
 
 $(eval $(call DEBUG_INFO_RULES, $(NVIDIA_MODPROBE)))
 $(NVIDIA_MODPROBE).unstripped: $(OBJS)
