diff -Naur nvidia-modprobe-550.70.old/Makefile nvidia-modprobe-550.70/Makefile
--- nvidia-modprobe-550.70.old/Makefile	2024-03-22 09:44:06.918286436 +0100
+++ nvidia-modprobe-550.70/Makefile	2024-03-22 09:44:17.502375236 +0100
@@ -110,7 +110,7 @@
 .PHONY: MANPAGE_install
 MANPAGE_install: $(MANPAGE)
 	$(MKDIR) $(MANDIR)
-	$(INSTALL) $(INSTALL_BIN_ARGS) $< $(MANDIR)/$(notdir $<)
+	$(INSTALL) $(INSTALL_DOC_ARGS) $< $(MANDIR)/$(notdir $<)
 
 $(eval $(call DEBUG_INFO_RULES, $(NVIDIA_MODPROBE)))
 $(NVIDIA_MODPROBE).unstripped: $(OBJS)
